### PR TITLE
Enable commitmentHash preclaim in tests

### PR DESCRIPTION
**Pull request by mradkov**

### CODE REVIEW

File: test/integration/contract.js [hunk 1]
```diff
@@ -215,12 +215,11 @@ describe('Contract', function () {
 
       // preclaim
       const { salt: _salt } = await contract.aensPreclaim(name)
```
Comment (marc0olo) [inline test/integration/contract.js:217]: the `aensPreclaim` should be replace with the contract call and delegation signature.

File: test/integration/contract.js [hunk 2]
```diff
@@ -215,12 +215,11 @@ describe('Contract', function () {
...
-      // @TODO enable after next HF
-      // const commitmentId = commitmentHash(name, _salt)
+      const commitmentId = commitmentHash(name, _salt)
```
Comment (marc0olo) [inline test/integration/contract.js:218]: the salt used here needs to be generated with the utils independent from the `aensPreclaim` method

**Review by marc0olo — CHANGES_REQUESTED** [review]

**Review by davidyuk — APPROVED** [review]

**Comment by marc0olo** [comment]
@mradkov I made some mistake. fixed it and squashed to commits into one. it's unfortunate to not see tests failing due to that linting and codecov errors

**Review by mradkov — COMMENTED** [review]
LGTM now

**Pull request merged**